libs/dahdi-linux/Makefile

```diff
@@ -27,6 +26,8 @@ PKG_EXTMOD_SUBDIRS:=drivers/dahdi
 
 include $(INCLUDE_DIR)/package.mk
 
+CONFIG_KERNEL_CFLAGS+="-Wno-old-style-declaration"
```

Comment:
Is this really necessary, cause for me on ARM64 and x86/64 its not falling to build without it